CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune-sklearn 0.1.3
+
+### Changes
+- Created pre-commit workflow triggered upon push
```

Comment:
> Created pre-commit workflow triggered upon push

I don't see this change in this commit.

Reply:
my bad, must have forgotten to commit changelog after the first update and so the two changes where combined into one commit